SCAutolib/models/gui.py

```diff
@@ -0,0 +1,355 @@
+from SCAutolib import run, logger
+
+import os
+from time import sleep, time
+import cv2
+import numpy as np
+import pytesseract
+import keyboard
+import uinput
+
+
+class Screen():
+    """Captures the screenshots."""
+
+    def __init__(self, directory: str):
+        """Init method
+        :param directory: Path to directory, where the screenshots
+            will be saved.
+        """
+        self.directory = directory
+        self.screenshot_num = 1
+
+    def screenshot(self, timeout: float = 30):
```

Comment:
param timeout should be described in the docstring

Reply:
ok